caps/import_utils.py

```diff
@@ -51,13 +49,15 @@ def add_authority_codes(filename):
     for index, row in plans_df.iterrows():
         council = row['council'].lower()
 
-        name_versions = [council, council.replace('council', '').strip(), council.replace('- unitary', '').replace('(unitary)', '').strip()]
+        name_versions = [council, council.replace('council', '').strip(
+        ), council.replace('- unitary', '').replace('(unitary)', '').strip()]
```

Comment:
one small request is that in future could formatting only changes and code changes be two separate commits?

Ideally I guess we should just take a one off formatting hit across the whole repo and then agree on formatting rules that we apply.

Reply:
Sorry I missed the autoformatter doing this. My defaults seems to be to use autopep8 with a few rules ignored. Could have a chat next week about if that's a good approach for a one off fix or if there's other rules to ignore?

Reply:
noiseworks look to be using black which seems like it gets round a lot of the "what rules do we use?" questions. I don't really mind what we use, as long as everyone is doing the same thing so we avoid reformatting wars :)

Reply:
Ooo, that looks quite nice.

Reply:
Yes, I think that would work.